src/confd/yang/confd.inc

```diff
@@ -43,7 +43,6 @@ MODULES=(
 	"ietf-netconf@2013-09-29.yang -e writable-running -e candidate -e rollback-on-error -e validate -e startup -e url -e xpath -e confirmed-commit"
 	"ietf-truststore@2023-12-28.yang -e central-truststore-supported -e certificates"
 	"ietf-keystore@2023-12-28.yang -e central-keystore-supported -e inline-definitions-supported -e asymmetric-keys -e symmetric-keys"
-	"ietf-ssh-server@2023-12-28.yang -e local-user-auth-password -e local-user-auth-publickey"
 	"ietf-tls-server@2023-12-28.yang -e server-ident-raw-public-key -e server-ident-x509-cert"
```

Comment:
This is the only part I'm slightly worried about, shouldn't it match the changes to netopeer2/scripts/common.sh?

Reply:
Could do, but not required. I think we should remove all that we do not need to change from that file. If we should add extra features it should of course remain.

Reply:
Yeah, valid point!  

Nice work on this, btw!